templates/rke2/overlay/opt/corral/rke2/proxy-env.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -ex
+
+if [ "$CORRAL_proxy_setup" = true ]; then
+#proxy settings for .bashrc 
+env="
+export HTTP_PROXY=http://${CORRAL_registry_private_ip}:3219
+export HTTPS_PROXY=http://${CORRAL_registry_private_ip}:3219
+export http_proxy=http://${CORRAL_registry_private_ip}:3219
+export https_proxy=http://${CORRAL_registry_private_ip}:3219
+export proxy_host=${CORRAL_registry_private_ip}:3219
```

Comment:
We already implemented half of the registry -> bastion rename output. Would this work if we used `bastion_private_ip` instead of `registry_private_ip`?